src/applications/disability-benefits/2346/sass/form-2346.scss

```diff
@@ -0,0 +1,31 @@
+@import '~@department-of-veterans-affairs/formation/sass/shared-variables';
+@import '~@department-of-veterans-affairs/formation/sass/modules/m-process-list';
+@import '~@department-of-veterans-affairs/formation/sass/modules/m-form-process';
+@import '~@department-of-veterans-affairs/formation/sass/modules/m-progress-bar';
+@import '../../../../platform/forms/sass/m-schemaform';
+@import '~@department-of-veterans-affairs/formation/sass/modules/m-modal';
+@import '~@department-of-veterans-affairs/formation/sass/modules/m-omb-info';
+@import '../../../../platform/forms/sass/m-form-confirmation';
+
+@mixin th-styles {
+  background-color: $color-gray-lightest;
+  font-weight: $font-bold;
+}
+.form2346 {
```

Comment:
Instead of adding custom styling, we should see if we can use any of our design utilities instead! 🙂 https://design.va.gov/utilities

Reply:
@mr0sari0 Can you take care of this please.

Reply:
@kelsonic @mojaray2k I removed this scss since we are not using any for our initial workflow.